src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1216,7 +1252,7 @@ public synchronized void afterDisconnect(SlaveComputer slaveComputer, TaskListen
                     if (sftpClient == null) {// system without SFTP
                         try {
                             connection.exec("rm " + fileName, listener.getLogger());
-                        } catch (Exception x) {
+                        } catch (Throwable x) {
```

Comment:
We got into several critical failures in buggy Jenkins extensions due to catching Exceptions and Missing throwables. Probably for this code it makes sense to accept Errors